src/seedu/addressbook/ui/Printable.java

```diff
@@ -0,0 +1,7 @@
+package seedu.addressbook.ui;
+
+public interface Printable {
```

Comment:
This interface and its methods should have header comments. If not, developers who need to inherit from this will not know how exactly they should override the methods and other developers who write client code for this will not know how to use it either.